site/content/documentation/reference/configuration.md

```diff
@@ -0,0 +1,458 @@
+---
+title: "Repository and SAIL configuration"
+toc: true
+weight: 4
+autonumbering: true
+---
+
+RDF4J repositories and SAIL configuration can be set up and changed by means of a configuration file in Turtle syntax. Here, we document the way the various components (repositories and SAILs) work together and how a full database configuration can be defined by "stacking" SAILs and wrapping in a Repository implementation.
```

Comment:
This is the actual new content. Everything else in the diff is mostly about the numbered sections and/or minor editorial updates.